build.sbt

```diff
@@ -167,9 +172,14 @@ import sbtassembly.AssemblyPlugin.defaultUniversalScript
 
 val scalacJvmOptions = Def.setting {
   val cross = CrossVersion.partialVersion(scalaVersion.value) match {
-    case Some((2, 13)) =>
-      Seq("-Ymacro-annotations", "-Xfatal-warnings", "-deprecation:false")
-    case _ => Seq.empty
+    case Some((2, 13)) => Seq(
+        "-Ymacro-annotations",
+        "-Xfatal-warnings",
+        "-deprecation:false",
+        "-target:8",
+        "-release:8",
+      )
+    case _ => Seq("-target:8", "-release:8")
```

Comment:
rather than adding these twice, to each scala version, should we follow the example of unused below and append separately?

Reply:
sure, done. I at some point I thought those might differ